builtin/common/tests/serialize_spec.lua

```diff
@@ -18,6 +18,18 @@ describe("serialize", function()
 		assert.same(test_in, test_out)
 	end)
 
+	it("handles precise numbers", function()
+		local test_in = math.random()
```

Comment:
A deterministic value here is better.
```suggestion
		local test_in = 0.269594915894577
```